### ISSUE TITLE
Doesn't train

### ISSUE THREAD

**Issue by wrrogers** [myoons/image-gpt-pytorch#1]
I've tried running the code and it doesn't train.  The loss is stuck at 1.8.  I've tried adjusting the learning rate between .001 and .00001, with no effect.

**Comment by myoons** [comment]
Sorry, the code is deprecated. I would update the code after work. Would you please try tomorrow?
